cip-CC-FEE-REMOVAL/cip-CC-FEE-REMOVAL.md

```diff
@@ -0,0 +1,276 @@
+<pre>
+  CIP:  CIP XXXX
+  Title: Canton Coin Fee Removal
+  Author:
+    Simon Meier
+  License: CC0-1.0
+  Status: Draft
+  Type: Tokenomics
+  Created: 2025-08-19
+  Approved:
+</pre>
+
+## Abstract
+
+This CIP proposes to remove Canton Coin fees and adjust holding fees so that
+users do not have to pay fees when transferring Canton Coin and application
+developers can build applications without special code to deal with these fees.
+This CIP further proposes to have users collect their validator rewards directly
+instead of indirectly via the validator operator party
+and to adjust the CC fees for creating or renewing `TransferPreapproval` contracts so
+that short-lived preapprovals (expiry < 90 days) can be created without paying CC fees.
+Thus making Canton Coin more attractive for both users and application
+developers.
```

Comment:
Note to reviewers: given recent discussions, it might be better to remove these two changes and land them as part of a later CIP.

Happy to do so.